src/pages/Index.vue

```diff
@@ -0,0 +1,26 @@
+<!--
+//  Index.vue
+//
+//  Created by Kalila L. on May 9th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<template>
+    <q-page class="flex flex-center">
+        <img
+            alt="Vircadia icon"
+            src="~assets/vircadia-icon.svg"
+        >
+    </q-page>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
```

Comment:
```suggestion
import { defineComponent } from "vue";
```
Single quotes are hard to distinguish from back-ticks.
Missing ";" makes it look like something's missing or code might continue on the next line.
Similarly for other .vue files.
This should be caught by ESLint rules once we fix them up.

Reply:
I'll add the semicolon rules in the next PR.